plugins/plugin-webpack/plugin.js

```diff
@@ -17,6 +18,23 @@ function insertBefore(newNode, existingNode) {
   existingNode.parentNode.insertBefore(newNode, existingNode);
 }
 
+function mountedRoots(mounts) {
+  const rootsAsUrls = Object.entries(mounts).map(([src, {url, static: isStatic}]) => ({
+    isStatic,
+    // Anything mounted at the root is "spread" into the output directory, so
+    // the individual files must be tracked rather than the destination
+    // directory as a whole.
+    output: url === '/' ? fs.readdirSync(src, 'utf-8').map((fileName) => `/${fileName}`) : [url],
+  }));
+
+  const staticRoots = rootsAsUrls.filter(({isStatic}) => isStatic).flatMap(({output}) => output);
+  const nonStaticRoots = rootsAsUrls
+    .filter(({isStatic}) => !isStatic)
+    .flatMap(({output}) => output);
+
+  return {staticRoots, nonStaticRoots};
```

Comment:
Currently only the `staticRoots` are used anywhere. However, since we have to partition out the `staticRoots` anyway, I figured we might as well make the `nonStaticRoots` available as well in case they're useful in the future.